src/videoosd.html

```diff
@@ -76,7 +76,7 @@ <h3 class="osdTitle"></h3>
 
                 <div class="volumeButtons hide-mouse-idle-tv">
                     <button is="paper-icon-button-light" class="buttonMute autoSize" title="${Mute} (m)">
-                        <i class="xlargePaperIconButton material-icons"></i>
+                        <span class="xlargePaperIconButton material-icons volume_up"></span>
```

Comment:
It was here. Then, we got double volume icon because of `iconElement.innerHTML = isMuted ? '&#xE04F;' : '&#xE050;';` and it was removed.
de28101f660a6f953b5971938b840791de24c5bc
Now this required to be here because of `classList.replace` - it requires "target".